EcoSoapBank/Network Controller/HTTPMethod.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  NetworkingEnums.swift
+//  EcoSoapBank
+//
+//  Created by Shawn Gee on 8/31/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+
+enum HTTPMethod: String {
+    case get = "GET"
+    case put = "PUT"
+    case post = "POST"
+    case patch = "PATCH"
+    case delete = "DELETE"
```

Comment:
I don't think this is necessary, all GraphQL calls are posts. Unless you have some other server that we're getting from.

Reply:
It's more just for having them available for any other networking that may need to be done outside of GraphQL.